Api/Endpoints/werkingsgebieden_data_manager.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2018 - 2022 Provincie Zuid-Holland
+
+from Api.Endpoints.data_manager import DataManager
+
+
+class WerkingsgebiedenDataManager(DataManager):
+
+    def _set_up_all_valid_view(self):
+        """
+        Set up a custom view that shows all valid version for werkingsgebieden
+        """
+        query = f"""
+                    CREATE OR ALTER VIEW {self.all_valid_view} AS
+                    SELECT
+                        w.ID,
+                        w.UUID,
+                        w.Begin_Geldigheid,
+                        w.Eind_Geldigheid,
+                        w.Created_Date,
+                        w.Modified_Date,
+                        w.Created_By,
+                        w.Modified_By,
+                        w.Werkingsgebied,
+                        w.symbol
```

Comment:
In the new views we do not fetch the `SHAPE` field which is of type geometry, as it is not supported by the mssql driver